reporters_db/data/reporters.json

```diff
@@ -5309,6 +5315,7 @@
                 "Dall.S.C.": "Dall.",
                 "Dallas": "Dall.",
                 "Dallas Rep.": "Dall.",
+                "U.S. (Dall.)": "Dall.",
                 "U.S.(Dall.)": "Dall."
```

Comment:
These two as well.  This isn't on you this is just what we did prior to this.

Reply:
Same question from review comment above.